portfolio/src/main/webapp/script.js

```diff
@@ -48,10 +40,8 @@ function showSlides(n) {
   }
 
   for (i = 0; i < slides.length; i++) {
-      slides[i].style.display = "none";
-  }
-  for (i = 0; i < dots.length; i++) {
-      dots[i].className = dots[i].className.replace(" active", "");
+    slides[i].style.display = "none";
+    dots[i].className = dots[i].className.replace(" active", "");
```

Comment:
I recommend adding parameter name comments. The style guide says:

“Parameter name” comments should be used whenever the value and method name do not sufficiently convey the meaning, and refactoring the method to be clearer is infeasible (see JS Practices advice). Their preferred format is before the value with "=":

someFunction(obviousParam, /* shouldRender= */ true, /* name= */ 'hello');

Reply:
done